README.md

```diff
@@ -1,11 +1,111 @@
-# ARES
-This repo has some code samples, documentation and usage of ARES test automation dashboard. This repo also has documentation on the usage, RESTAPI and other details
+<p><img src="https://i.ibb.co/kSgVFJF/ARES.png" width="300" height="150"></p>
 
-For more info on the tutorials, documentation, please refer to our wiki https://github.com/testastra/ARES/wiki
+# ARES DASHBOARD
 
-### Available Sample Code in 
+> ARES is a web based portal, which provides automation results at one place. It can integrate with most popular automation tools like Selenium, Protractor etc. This repo consist some samples and usage of ARES test automation dashboard
 
-* Java
-* Java Script
-* C#
-* Python (coming soon...)
+![HitCount](http://hits.dwyl.io/testastra/ares.svg)
+![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)
+![Open Source Love png1](https://badges.frapsoft.com/os/v1/open-source.png?v=103)
+[![License](https://img.shields.io/badge/license-GPLv3-blue.svg)](http://www.gnu.org/licenses/gpl-3.0.html)
+[![Ask Me Anything !](https://img.shields.io/badge/Ask%20me-anything-1abc9c.svg)](mailto:info@testastra.com?Subject=ARES%20Dashboard)
+![ForTheBadge built-with-love](http://ForTheBadge.com/images/badges/built-with-love.svg)
+
+***ARES DASHBOARD OVERVIEW***
+
+![ARES DASHBOARD](http://testastra.com/assets/images/header_center.png)
+
+---
+__KEY FEATURES__
+
+- Live Execution status
+- Works with any automation tool (by using RESTAPI calls)
+- Supports Historical results
+- Visualization of execution status using charts
+  - Test Automation Health Status
+  - Test Automation Execution Trends
+  - Top 5 failed test cases
+  - Test Execution Time Trends
+- Centralized report (view reports from any part of globe by login)
+
+---
+__Why ARES__
+
+- It's Open Source with unlimited features
+- No Setup or Installation required
+- Made by QA
+
+---
+__SAMPLE REPORT__
+
+ - Sample Report [link](https://www.youtube.com/watch?v=KiQDBw9kAHI&feature=youtu.be)
```

Comment:
Right now, we just have only one youtube link for our webinar, and currently no sample reports available. we can publish few public and can add those references later here. For now, lets keep youtube reference only

Reply:
Included only one link